src/server/chromium/crPage.ts

```diff
@@ -848,18 +848,12 @@ class FrameSession {
     const ffmpegPath = registry.findExecutable('ffmpeg')!.executablePath();
     // TODO: use default error message once it's ready.
     if (!ffmpegPath || !canAccessFile(ffmpegPath)) {
```

Comment:
Why don't we use executablePathOrDie() here? We can append `Please install ffmpeg in order to record video.` to the error, or pass custom text to `executablePathOrDie` instead. It already has a nice `<3 Playwright Team` message.

Reply:
good catch! one less exported function.